examples/database/src/lib.rs

```diff
@@ -13,6 +13,9 @@ pub fn hello<R: Database<Json>>(rt: &mut R) {
 }
 
 #[rock_n_rollup::main]
-pub fn main<R: Runtime>(application: &mut Application<R>) {
+pub fn main<R>(application: &mut Application<R>)
+where
+    R: rock_n_rollup::core::Runtime,
+{
```

Comment:
I prefer the old syntax, which seems easier.

I never know in Rust, if where should be used or not.
Basically, when there is only one trait and one generic parameter, I prefer not using it.

Reply:
I change to use the old way 
```
pub fn main<R:rock_n_rollup::core::Runtime>...
```
and remove the `where`

it has been changed at other files as well at this [commit](3d56b448ce27ff2ee25b9e46e3b86aecbbb74896)